.github/workflows/pull_request_check.yml

```
@@ -0,0 +1,25 @@
+name: Pull Request Check
+
+on:
+  pull_request:
+    branches:
+      - alpha
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)